### PR TITLE
chore(TS): export types

### PR DESCRIPTION
**Pull request by ShaMan123**
## Changes

A few more export types

### CODE REVIEW

**Review by ShaMan123 — COMMENTED** [review]
looks ready

**Review by ShaMan123 — COMMENTED** [review]
Now is the time to decide about naming
You said you do not like `Props` and prefer `Options`: `CircleProps` => `CircleOptions`
I am fine with that

**Comment by jiayihu** [comment]
`CircleOptions` is more consistent with the JS variable name indeed

File: src/util/index.ts [hunk 1]
```diff
@@ -32,6 +39,8 @@ export {
   multiplyTransformMatrices,
   multiplyTransformMatrixArray,
   isIdentityMatrix,
+  angleToSkew,
+  skewToAngle,
```
Comment (asturur) [inline src/util/index.ts:43]: ae those useful?

Comment (ShaMan123) [reply src/util/index.ts:43]: Do not know
But might be
Will save a dev frim reading about skew
Not important for me

**Comment by ShaMan123** [comment]
@asturur change this PR as you wish and merge when possible
I am not attached to anything apart from our long lasting buf about `T` prefixing 🤫🤐

**Comment by ShaMan123** [comment]
> `CircleOptions` is more consistent with the JS variable name indeed

I think we should do it in a separate PR - it is huge diff (34 files)

File: src/util/index.ts (hunk 1 above)
Comment (asturur) [reply src/util/index.ts]: I removed them for now, when we have time to put a description over those and explain what they do and what is the value returned useful for, we can expose them.

**Comment by asturur** [comment]
Yes in general i don't like that we call PROPS everything than before was options because of... react i suppose.
But let's expose the types, and circle back on that if we want

**Review by asturur — APPROVED** [review]

**Pull request merged**